index.html

```diff
@@ -125,18 +125,22 @@ <h2>MiniApp Global Application Lifecycle Events</h2>
             <p>MiniApp <dfn data-dfn-for="Document" data-export="" data-dfn-type=
           "dfn">in error</dfn></p>
           </li>
+          <li>
+            <p>MiniApp <dfn data-dfn-for="Document" data-export="" data-dfn-type=
+          "dfn">unloading</dfn></p>
+          </li>
           </ul>
           
           <p>For each MiniApp, after [=Document/initialization=], it will be either [=Document/running in foreground=] or [=Document/running in background=].</p>
           <p>When user chooses to close the MiniApp by clicking the close button on MiniApp, or go to the mobile phone’s home screen, the MiniApp will not be destroyed immediately, but switch to be [=Document/running in background=].</p>
           <p>When user reopens the same MiniApp, MiniApp will switch from [=Document/running in background=] to [=Document/running in foreground=].</p>
-          <p>Only when MiniApp runs in the background for more than a specific time duration (e.g., 5 minutes), or occupies too much system resources in the background, the MiniApp will be destroyed.</p>
+          <p>[=Document/unloading=] marks the end of the MiniApp session, and the removal of all the temporary resources from the cache. MiniApp user agents perform this removal once the MiniApp meets criteria. By default, when MiniApp runs in the background for more than a specific time duration (e.g., 5 minutes), or occupies too much system resources in the background, the MiniApp will be unloaded. </p>
```

Comment:
I suggest that we change "by default" to an RFC 2119 key word "SHOULD" or "MAY".